images/plbase/plbase-install.sh

```diff
@@ -10,31 +10,32 @@ amazon-linux-extras install -y \
     redis4.0
 
 # Notes:
-# - `libjpeg-devel` is needed by the Pillow package
 # - `gcc-c++` is needed to build the native bindings in `packages/bind-mount`
-# `texlive` and `texlive-dvipng` are needed for matplotlib LaTeX labels
+# - `libjpeg-devel` is needed by the Pillow package
+# - `procps-ng` is needed for the `pkill` executable, which is used by `zygote.py`
+# - `texlive` and `texlive-dvipng` are needed for matplotlib LaTeX labels
 yum -y install \
-    tmux \
-    postgresql-server \
-    postgresql-contrib \
-    man \
+    dos2unix \
     emacs-nox \
     gcc \
-    make \
+    gcc-c++ \
+    git \
+    graphviz \
+    graphviz-devel \
+    ImageMagick \
+    libjpeg-devel \
     lsof \
+    make \
+    man \
     openssl \
-    dos2unix \
+    postgresql-contrib \
+    postgresql-server \
+    procps-ng \
```

Comment:
This is the only addition; the rest of the entries in this list were just alphabetized.